src/scenes/InternalScene.ts

```diff
@@ -17,72 +19,149 @@ export default class InternalScene extends BaseScene {
                 // do the bateria
                 console.log("...The BACTERIA. which was found in the number check");
-                this.microGuy = this.art.bacteria;
-                this.art.bacteria.visible = true;
+                
                 break;
             case MICROORGANISM_INDEX.BACTERIA_PROTEIN:
                 console.log("...The BACTERIA_PROTEIN which was found in the number check");
-                this.microGuy = this.art.protein;
-                this.art.protein.visible = true;
+                
                 break;
             case MICROORGANISM_INDEX.DEAD_VIRUS:
                 break;
             case MICROORGANISM_INDEX.LIVE_VIRUS:
                 break;            
         }
 
-        this.art.bacteria.gotoAndStop(0);
-        this.art.protein.gotoAndStop(0);
+        // this.art.bacteria.gotoAndStop(0);
+        // this.art.protein.gotoAndStop(0);
         Utils.simpleButton(this.art.back);
 
     }
 
     start(){
-        //
-        PIXI.animate.Animator.play(this.microGuy,"intro",this.boingIt);
+        // PIXI.animate.Animator.play(this.microGuy,"intro",this.boingIt);
 
         this.art.back.once("pointerdown", ()=>{
             this.changeScene("game");
         });
-    }
 
-    boingIt = () => {
-        // 
-        Tween.get(this.microGuy)
-            .to({x:this.art.back.x + this.art.back.width + this.microGuy.width,y:this.microGuy.height},800,"quadOut")
-            .to({y:this.stageManager.height-this.microGuy.height/2},1300,"bounceOut")
-            .call(this.endIt);
-    }
-    endIt = () => {
-        PIXI.animate.Animator.play(this.microGuy,"reveal",this.boingIt);
+
+
+        //initialize arrays
+
     }
 
+    // boingIt = () => {
+    //     // 
+    //     Tween.get(this.microGuy)
+    //         .to({x:this.art.back.x + this.art.back.width + this.microGuy.width,y:this.microGuy.height},800,"quadOut")
+    //         .to({y:this.stageManager.height-this.microGuy.height/2},1300,"bounceOut")
+    //         .call(this.endIt);
+    // }
+    // endIt = () => {
+    //     PIXI.animate.Animator.play(this.microGuy,"reveal",this.boingIt);
+    // }
+
     update(){
+        this.macrophageArray.forEach(macrophage => {
+            this.bacteriaArray.forEach(bacteria => {
+                if(!bacteria.isDividing&&!macrophage.isHit) {
+                    macrophage.isHit = 
+                    bacteria.x < macrophage.x + macrophage.width/2 && 
+                    bacteria.x > macrophage.x - macrophage.width/2  &&
+                    bacteria.y > macrophage.y - macrophage.height/2 &&
+                    bacteria.y < macrophage.y + macrophage.height/2;
+                    if(macrophage.isHit) {
+                        PIXI.animate.Animator.play(macrophage, "dead", this.checkMacrophage);
+                    }
+                }
+                
+            });
+        });
+
+        this.macrophageArray.forEach(macrophage => {
+            macrophage.gotoAndStop(0);
```

Comment:
remove this line, because this is keeping the frame at 0 every update tic, and overriding the play function on hit!